packages/core/utils/src/check.js

```diff
@@ -0,0 +1,62 @@
+const WebSocket = require("ws");
```

Comment:
Put this in Utils instead of duplicating in all Clients.
When we refactor the others, we can use this instead (they all use the same functions)